test/unit/lib/country-data.test.js

```diff
@@ -0,0 +1,108 @@
+const {
+    countryOptions,
+    overrideLabels,
+    dupeCommonOptions,
+    registrationCountryOptions,
+    subdivisionOptions
+} = require('../../../src/lib/country-data');
+
+describe('unit test lib/country-data.js', () => {
+
+    test('countryOptions has the ballpark number of countries we expect, in alphabetical order', () => {
+        expect(typeof countryOptions).toBe('object');
+        expect(countryOptions.length > 200).toEqual(true);
+        expect(countryOptions.length < 300).toEqual(true);
+        const sortedCountryOptions = countryOptions.sort((a, b) => {
+            if (a.label < b.label) {
+                return -1;
+            }
+            return 1;
+        });
+        expect(countryOptions.map(option => option.label))
```

Comment:
With this last test, are you trying to test that re-sorting doesn't change the order to prove that it is already sorted?

Reply:
yes. i'll add a comment to make that clearer.